src/main/kotlin/racing/view/ResultView.kt

```diff
@@ -0,0 +1,30 @@
+package racing.view
+
+import racing.domain.Car
+
+object ResultView {
+    private const val PROGRESS_STRING = "-"
+    private const val SEPERATOR = ", "
+    
+    fun showWinner(cars: List<Car>) {
+        val firstGradePosition = cars.maxOf(Car::position)
```

Comment:
람다로 사용할 수 있어요~!

참조와 람다의 차이와 어떤게 더 나은 방법일지는 아래의 문서를 참고해주세요 :) 

https://dev.to/vtsen/prefer-function-reference-over-lambda-in-kotlin-wrong-1f8k?fbclid=IwAR06TovgOmNn-1zAejUa5RgYY8tAwhMv0vV1J971PFyi7GZEaUP0Zz8gKtc

```suggestion
        val firstGradePosition = cars.maxOf { it.position }
```

Reply:
공유주신 링크 확인해 보도록 하겠습니다.